analyzer/src/main/kotlin/managers/Conan.kt

```diff
@@ -145,7 +145,7 @@ class Conan(
             conanConfig?.also { configureRemoteAuthentication(it) }
 
             val jsonFile = createOrtTempDir().resolve("info.json")
-            run(workingDir, "info", ".", "--json", jsonFile.absolutePath, *DUMMY_COMPILER_SETTINGS)
+            run(workingDir, "info", definitionFile.name, "--json", jsonFile.absolutePath, *DUMMY_COMPILER_SETTINGS)
```

Comment:
Hmm, won't this code now always use `conanfile.txt` if both `conanfile.txt` and `conanfile.py` are present? Because our `globsForDefinitionFiles` are prioritized, and we list `conanfile.txt` before `conanfile.py`.

There's probably nothing wrong with this, as it makes the `conan info` call aligned with the semantics of `globsForDefinitionFiles`. But if that's what you intended, I find the commit message a bit misleading.

Reply:
Yes, if both are present `PackageManager.findManagedFiles()` will only keep the first file, in this case "conanfile.txt". However, from the point of view of the function this does not matter, it is only important, that it handles the passed `definitionFile` correctly. Programmatically you could call `resolveDependencies(definitionFile: File, ...)` for any file.